lib/server.js

```diff
@@ -108,10 +108,10 @@ Server.prototype._publish = function() {
   var bridgeConfig = this._config.bridge || {};
 
   var info = this._bridge.getService(Service.AccessoryInformation);
-  info.setCharacteristic(Characteristic.Manufacturer, `${toTitleCase(require('../package.json').author.name)}`);
-  info.setCharacteristic(Characteristic.Model, `${toTitleCase(require('../package.json').name)}`);
+  info.setCharacteristic(Characteristic.Manufacturer, bridgeConfig.manufacturer || `${toTitleCase(require('../package.json').author.name)}`);
+  info.setCharacteristic(Characteristic.Model, bridgeConfig.model || `${toTitleCase(require('../package.json').name)}`);
   info.setCharacteristic(Characteristic.SerialNumber, bridgeConfig.username);
-  info.setCharacteristic(Characteristic.FirmwareRevision, require('../package.json').version);
+  info.setCharacteristic(Characteristic.FirmwareRevision, `${toTitleCase(require('../package.json').version)}`);
```

Comment:
Why toTitleCase here?

Reply:
I was just copy and pasting from the top and changing it to firmware. I will revert it back.